lib/std/build.zig

```diff
@@ -1710,7 +1710,7 @@ pub const LibExeObjStep = struct {
                 inline for (enum_info.fields) |field| {
                     out.print("    {},\n", .{field.name}) catch unreachable;
                 }
-                out.print("}};\n", .{}) catch unreachable;
+                out.print("{};\n", .{"}"}) catch unreachable;
```

Comment:
```suggestion
                out.print("};\n", .{}) catch unreachable;
```

Wouldn't this work as well?

Reply:
```suggestion
                out.writeAll("};\n") catch unreachable;
```

Or maybe not using `format` at all